frontend/angular.json

```diff
@@ -0,0 +1,123 @@
+{
```

Comment:
use tabs and the indent should be different... { on new line... I will extend the eslint ruleset for an autofix